umake/ui/cli/__init__.py

```diff
@@ -157,6 +157,60 @@ def mangle_args_for_default_framework(args):
     return result_args
 
 
+def list_frameworks(listing, detailled=True):
+    if listing == "all" or listing == "available":
```

Comment:
I understand your idea of return everything and sort the output after. So I removed the listing and detailled arguments.